backend/private-graph/graph/schema.resolvers.go

```diff
@@ -6224,14 +6224,12 @@ func (r *queryResolver) Admin(ctx context.Context) (*model.Admin, error) {
 			Phone:                 &firebaseUser.PhoneNumber,
 			AboutYouDetailsFilled: &model.F,
 		}
-		if err := r.DB.Create(newAdmin).Error; err != nil {
+		if err := r.DB.Where(newAdmin).FirstOrCreate(&admin).Error; err != nil {
```

Comment:
I was hitting an issue where there was only 1 network request I could find hitting this method, but somehow it was being invoked 3 times and trying to call `Create` with the same attributes more than once. I'm still not sure what the culprit is there, but I figured this change couldn't hurt to make it more resilient if it gets called quickly in succession.